drivers/meross-smart-wifi-garage-door-opener.groovy

```diff
@@ -97,6 +97,10 @@ def refresh() {
             body: '{"payload":{},"header":{"messageId":"'+settings.messageId+'","method":"GET","from":"http://'+settings.deviceIp+'/config","sign":"'+settings.sign+'","namespace": "Appliance.System.All","triggerSrc":"iOSLocal","timestamp":' + settings.timestamp + ',"payloadVersion":1}}'
         ])
         log hubAction
+        runIn(20, refresh, [overwrite: false])
```

Comment:
wouldn't refresh calling itself return an endless loop of constant refreshes? Wondering if it would make more sense to call these in `open` and `close`

Reply:
Being in the SendCommand section rather than inside the refresh loop, I seems to only be executed when you send an open/close command. The refresh did not make use of the sendcommand().
It’s a single refresh request, to check in 10, 20, and again 30 seconds. (My garage takes ~22 seconds to open).
Since the overwrite is false, it does not interfere with the standard schedule of every 5 minutes.
I watched the schedule and so far it has worked well for me.
But if SendCommand() is being used in refresh then absolutely right :)